src/views/List.jsx

```diff
@@ -12,28 +13,41 @@ export function List({ data }) {
 		item.name.toLowerCase().includes(searchedItem.toLowerCase()),
 	);
 
+	let navigate = useNavigate();
+	const routeToAddItem = () => {
+		let path = `/add-item`;
+		navigate(path);
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
-			<form onSubmit={(e) => e.preventDefault()}>
-				<label htmlFor="search">Filter items</label>
-				<input
-					id="search"
-					type="text"
-					placeholder="Start typing here..."
-					value={searchedItem}
-					onChange={handleChange}
-				/>
-				{searchedItem && (
-					<button type="reset" onClick={() => setSearchedItem('')}>
-						X
-					</button>
-				)}
-			</form>
+			{!data.length ? (
+				<div>
+					<p>Your shopping list is currently empty.</p>
+					<button onClick={routeToAddItem}>Add your first item</button>
+				</div>
+			) : (
+				<form onSubmit={(e) => e.preventDefault()}>
+					<label htmlFor="search">Filter items</label>
+					<input
+						id="search"
+						type="text"
+						placeholder="Start typing here..."
+						value={searchedItem}
+						onChange={handleChange}
+					/>
+					{searchedItem && (
+						<button type="reset" onClick={() => setSearchedItem('')}>
+							X
+						</button>
+					)}
+				</form>
+			)}
```

Comment:
Stylistically, I'm not sure if I'd use such large blocks in a ternary expression. But that's just me :)